index.html

```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
           </ul>
         </div>
         <div class="space"></div>
+        <div>
+          <img
+            class="drop-down"
+            src="Images/drop-down.png"
+            alt="drop-down button"
+          />
+        </div>
+      </section>
+    </main>
+    <section class="rest">
```

Comment:
[OPTIONAL] I think this section should be within your main tag. The idea would be that:
- All your code apart from nav and footer will be in the main tag
- Within the main ag you have different section tags
- One section will be for the landing page
- Another section for recent works
- And another section for about me